client/tests.py

```diff
@@ -0,0 +1,5 @@
+"""Tests for application."""
+
+from django.test import TestCase
+
+# Create your tests here.
```

Comment:
Write tests for all api endpoints.

Reply:
https://wsvincent.com/django-testing-tutorial/
https://www.django-rest-framework.org/api-guide/testing/#apiclient
use those to create tests.